enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -539,6 +539,7 @@ type FirecrackerContainer struct {
 	env                environment.Env
 	mountWorkspaceFile bool
 
+	netns           *networking.Namespace
```

Comment:
nit: Rename to networkingNamespace? To make a bit more readable

Reply:
went with [network namespace](https://man7.org/linux/man-pages/man7/network_namespaces.7.html)